common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -18,11 +18,15 @@
 import org.springframework.data.domain.Sort.Direction;
 import org.springframework.data.jpa.domain.Specification;
 import org.springframework.stereotype.Service;
-import org.pmiops.workbench.model.SearchType;
 
 @Service
 public class ConceptService {
 
+    public static enum SearchType
+    {
```

Comment:
nit: normally this `{` would go on the previous line, as with a class. We should have a more obvious style guide so this isn't so subjective though..